RELEASE.md

```diff
@@ -1,3 +1,83 @@
+* Adding support for datasets as inputs to `from_tensors` and `from_tensor_slices` and batching and unbatching of nested datasets.
+* Add HW acceleration support for topK_v2
+* Deprecated `tf.keras.experimental.export_saved_model` and `tf.keras.experimental.function`. Please use `tf.keras.models.save_model(..., save_format='tf')` and `tf.keras.models.load_model` instead.
+* Add new `TypeSpec` classes
+* CloudBigtable version updated to v0.10.0 BEGIN_PUBLIC CloudBigtable version updated to v0.10.0
+* Deprecated the use of `constraint=` and `.constraint` with ResourceVariable.
+* Expose Head as public API.
+* AutoGraph is now applied automatically to user functions passed to APIs of `tf.data` and `tf.distribute`. If AutoGraph is disabled in the the calling code, it will also be disabled in the user functions.
+* Update docstring for gather to properly describe the non-empty batch_dims case.
+* Added `tf.sparse.from_dense` utility function.
+* Add `GATHER` support to NN API delegate
+* AutoGraph translates Python control flow into TensorFlow expressions, allowing users to write regular Python inside `tf.function`-decorated functions. AutoGraph is also applied in functions used with `tf.data`, `tf.distribute` and `tf.keras` APIs.
+* Improved ragged tensor support in `TensorFlowTestCase`.
+* Makes the a-normal form transformation in Pyct configurable as to which nodes are converted to variables and which are not.
+* `keras.backend.resize_images` (and consequently, `keras.layers.Upsampling2D`) behavior has changed, a bug in the resizing implementation was fixed.
+* `ResizeInputTensor` now works for all delegates
+* Start of open development of TF, TFLite, XLA MLIR dialects.
+* Add `EXPAND_DIMS` support to NN API delegate TEST:  expand_dims_test
+* `tf.cond` emits a StatelessIf op if the branch functions are stateless and do not touch any resources.
+* Add support of local soft device placement for eager op.
+* Pass partial_pivoting to the `_TridiagonalSolveGrad`.
+* Add HW acceleration support for `LogSoftMax`.
+* Added a function nested_value_rowids for ragged tensors.
+* fixed a bug in histogram_op.cc.
+* Add guard to avoid acceleration of L2 Normalization with input rank != 4
+* Added evaluation script for COCO minival
+* Add delegate support for QUANTIZE
+* tflite object detection script has a debug mode
+* Add `tf.math.cumulative_logsumexp operation`.
+* Add `tf.ragged.stack`.
+* Add delegate support for `QUANTIZED_16BIT_LSTM`.
+* Add an `implementation=3` mode for `tf.keras.layers.LocallyConnected2D` and `tf.keras.layers.LocallyConnected1D` layers using `tf.SparseTensor` to store weights,  allowing a dramatic speedup for large sparse models.
+* `tf.while_loop` emits a StatelessWhile op if the cond and body functions are stateless and do not touch any resources.
```

Comment:
would be nice to group these better: this one should go next to the corresponding tf.cond change above.

Reply:
Done.